quickwit/quickwit-serve/src/debugging_api.rs

```diff
@@ -17,41 +17,102 @@
 // You should have received a copy of the GNU Affero General Public License
 // along with this program. If not, see <http://www.gnu.org/licenses/>.
 
+use std::sync::Arc;
+
+use quickwit_ingest::Ingester;
 use quickwit_proto::control_plane::{
     ControlPlaneService, ControlPlaneServiceClient, GetDebugStateRequest,
 };
+use warp::{Filter, Rejection};
+
+use crate::{with_arg, QuickwitServices};
 
 #[derive(utoipa::OpenApi)]
-#[openapi(paths(debugging_handler))]
+#[openapi(paths(control_plane_debugging_handler, mrecordlog_debugging_handler,))]
 /// Endpoints which are weirdly tied to another crate with no
 /// other bits of information attached.
 ///
 /// If a crate plans to encompass different schemas, handlers, etc...
 /// Then it should have it's own specific API group.
 pub struct DebugApi;
 
+pub(crate) fn debugging_routes(
+    quickwit_services: Arc<QuickwitServices>,
+) -> impl Filter<Extract = (impl warp::Reply,), Error = Rejection> + Clone {
+    let debugging_routes = warp::path!("api" / "debugging" / ..);
+
+    debugging_routes.and(
+        control_plane_debugging_handler(quickwit_services.control_plane_service.clone()).or(
+            mrecordlog_debugging_handler(quickwit_services.ingester_opt.clone()),
+        ),
+    )
+}
+
 #[utoipa::path(
     get,
-    tag = "Get debug information for node",
-    path = "/",
+    tag = "Get debug informations about the control plane from the node viewpoint",
+    path = "/shard",
```

Comment:
I'm not sure about the path `/shard`.

Note that your description one line above suggests that the name is not great.
Could we call it `indexing_plan`  or `control_plane` or something like that?